src/services/qix/index.js

```diff
@@ -111,10 +114,24 @@ export default class Qix {
       url += `/identity/${encodeURIComponent(identity)}`;
     }
 
+    let urlParamKeys;
+
     if (reloadURI) {
       url += `?reloadUri=${encodeURIComponent(reloadURI)}`;
     }
 
+    if (urlParams) {
+      urlParamKeys = Object.keys(urlParams);
+      if (url.indexOf('?') === -1) {
+        url += '?';
+      }
+      for (let i = 0; i < urlParamKeys.length; i += 1) {
+        url += `${urlParamKeys[i]}=${encodeURIComponent(urlParams[urlParamKeys[i]])}`;
```

Comment:
if there are multiple urlParams they needs to be separated by a &
it would also be a good idea to encode the parameter name